src/React.js

```diff
@@ -65,5 +65,6 @@ assign(React, {
 });
 
 React.__SECRET_DOM_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = ReactDOM;
+React.__SECRET_DOM_SERVER_DO_NOT_USE_OR_YOU_WILL_BE_FIRED = ReactDOMServer;
```

Comment:
The point is that the `react-dom-server.js` package we provide uses this secret key, but you're meant to never type it in your own code.